### DISCUSSION TITLE
What is the traditional "Baked Alaska" dessert made of?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#875] [Galaxy Trivia] [answered]
**Bonus:** Describe the steps how to make one.

**Comment by peterbaccaro** [comment, marked as answer]
The traditional dessert "Baked Alaska" is **made of a layer of cake topped with ice cream and covered in a layer of meringue**.

**Ingredients:**

- Cake base (sponge cake or pound cake)
- Ice cream (flavor of your choice)
- Meringue (made from egg whites and sugar)

**Steps:**

- Prepare the cake: Start by baking or obtaining a cake base. You can use store-bought or homemade sponge cake or pound cake. Allow the cake to cool completely before proceeding.

- Shape the cake: Trim the cake into the desired shape and size. You can make individual portions or a larger cake for multiple servings.

- Add the ice cream: Place a layer of plastic wrap or parchment paper in the desired mold or dish to help with easy removal. Place the cake layer in the bottom of the mold. Next, spread a layer of ice cream on top of the cake. You can use a single flavor or multiple flavors of ice cream.

- Freeze the assembled cake and ice cream: Once the ice cream layer is added, place the dessert in the freezer and let it freeze solid. This will help ensure that the ice cream remains firm during the baking process.

- Prepare the meringue: In a clean, dry bowl, whisk the egg whites until they become frothy. Gradually add sugar while continuing to whisk until the mixture forms stiff peaks. The meringue should be glossy and smooth.

- Apply the meringue: Remove the frozen cake and ice cream from the freezer. Cover the entire dessert with the meringue, making sure to seal it completely. You can use a spatula or a piping bag to create decorative patterns.

- Bake or torch: Preheat your oven to a high temperature (around 425°F/220°C) or prepare a kitchen torch. If using the oven, carefully place the Baked Alaska on a baking sheet and bake for a short period, usually around 3-5 minutes, or until the meringue is lightly browned. If using a torch, quickly and evenly brown the meringue by moving the flame over the surface.

- Serve immediately: Once the meringue is browned, remove the Baked Alaska from the oven or torch and serve it immediately. Cut into slices or individual portions and enjoy the contrast of textures and flavours.
